demos/projects/NXP/mimxrt1060/VSCodeDebug.md

```diff
@@ -0,0 +1,69 @@
+# Debug an NXP MIMXRT1060-EVK Evaluation kit using VS Code
+
+## Before you start
+
+In order to debug the MIMXRT1060-EVK with Visual Studio, this guide requires Segger J-Link. While it works, currently this  experimental and [not suported scenario](https://forum.segger.com/index.php/Thread/8140-SOLVED-CMSIS-DAP-support-seems-to-be-disabled/?postID=29869) for Segger, so try it at your own risk.  
+
+## What you need
+
+* Successfully build the sample for the NXP Evaluation Kit DevKit: [MIMXRT1060-EVK](https://www.nxp.com/design/development-boards/i-mx-evaluation-and-development-boards/mimxrt1060-evk-i-mx-rt1060-evaluation-kit:MIMXRT1060-EVK).
+
+* Make sure to run this git command before building and cloning:
+
+ ```powershell
+ # As **administrator** 
+ git config --system core.longpaths true
+```
```

Comment:
this bullet should go first, since the above is actually building. you might get the error on build and then see this bullet